### DISCUSSION TITLE
Heading 3

### DISCUSSION THREAD

**Discussion by jeffreylauwers** [nl-design-system/backlog#319] [Component suggestions]
<img width="720" alt="visuele weergave van de heading 3 component" src="https://github.com/user-attachments/assets/7afe7f58-daee-4eb5-a5dd-809be32bbe7d">

## Naam

Heading 3

## Doel

Koptekst die in de koppenstructuur op het derde niveau staat.

## Ook bekend als

Heading, Titel

## Gerelateerde componenten

[Heading](https://github.com/orgs/nl-design-system/discussions/220), [Heading 1](https://github.com/orgs/nl-design-system/discussions/317), [Heading 2](https://github.com/orgs/nl-design-system/discussions/318), [Heading 4](https://github.com/orgs/nl-design-system/discussions/320), [Heading 5](https://github.com/orgs/nl-design-system/discussions/321), [Heading 6](https://github.com/orgs/nl-design-system/discussions/322), [Rich Text Content](https://github.com/orgs/nl-design-system/discussions/289)

## Onderzoek

**Comment by Yolijn** [comment]
## ✨ Dit component is nu Community ✨

Help je mee hem door de Candidate stappen te krijgen?
[Heading 3 op het Candidate bord](https://github.com/orgs/nl-design-system/projects/32/views/1?filterQuery=heading+3)
